portal-parent/ozwillo-portal-front/pom.xml

```diff
@@ -113,6 +119,12 @@
             <artifactId>memcached-session-manager-tc8</artifactId>
         </dependency>
 
+        <!-- Lombok -->
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
```

Comment:
Lombok is under use at this moment, but i want clean some class like [this](https://github.com/ozwillo/ozwillo-portal/blob/master/portal-parent/ozwillo-portal-front/src/main/java/org/oasis_eu/portal/model/notifications/UserNotification.java).
It's an [ugly hack when inject generated code in Eclipse](https://www.reddit.com/r/programming/comments/9bb1e/project_lombok_automatic_resource_management_for/c0c5f7o) (i doesn't know for IntelliJ) but useful to make more readable class with tons of boilerplate code.